### PR TITLE
Add example for specifying package version

### PR DESCRIPTION
**Pull request by hnrkndrssn**
Also an update to the description for the field to mention not to use single-quotes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,23 @@ steps:
...
+  OCTOPUS_URL: ${{ secrets.OCTOPUS_URL }}
+  OCTOPUS_SPACE: 'Outer Space'
+steps:
+  # ...
+  - name: Create a release in Octopus Deploy 🐙
+    uses: OctopusDeploy/create-release-action@v3
+    with:
+      project: 'MyProject'
+      packages: |
+        StepName:PackageReferenceName:Version
```
Comment (benPearce1) [inline README.md:71]: would it be worth having some additional variations here:

```
packages: |
    StepName:PackageReferenceName:Version
    PackageReferenceName2:Version2
```

Or possibly a note that says for additional packages simply to add another line under the `packages` node, and that the step name is optional.

**Comment by geofflamrock** [comment]
> Also an update to the description for the field to mention not to use single-quotes

I might be missing something, but I can't see the above change?

**Comment by hnrkndrssn** [comment]
> > Also an update to the description for the field to mention not to use single-quotes
> 
> I might be missing something, but I can't see the above change?

ha, helps if I saved the change before committing 🤦

File: README.md (hunk 1 above)
Comment (hnrkndrssn) [reply README.md:71]: I've added another entry to the parameter

**Review by benPearce1 — APPROVED** [review]

**Review by geofflamrock — APPROVED** [review]
LGTM, thanks for improving the docs!

**Pull request merged**